tools/explorer/lib/tests/test_constants.py

```diff
@@ -0,0 +1,10 @@
+"""Constants for the ontology explorer application."""
+
+from os import path
+
+# TODO(b/183433133):move to one constants file after refactoring.
+JOINS = path.join('..', '..', '..', '..')
```

Comment:
you can reuse this one instead of defining a new one?
https://github.com/google/digitalbuildings/blob/master/tools/validators/instance_validator/validate/constants.py

Reply:
ah maybe not? it is in a different folder

Reply:
Yeah I think not. I've tried using the ontology validator constants before and it's easier to just create my own constants since the explorer is a separate package.